iree/compiler/Codegen/LLVMCPU/KernelDispatch.h

```diff
@@ -23,6 +23,16 @@ enum class TilingLevel : unsigned {
   NumTileLevels = 3
 };
 
+enum class StrategyTilingLevel : unsigned {
```

Comment:
Doesnt that replace the `TilingLevel` class above?

Reply:
The TilingLevel is still used in ARM CPU pipeline. I think it's still meaningful for mmt4d and CPUTileFuseAndVectorize approach.